src/core/main.c

```diff
@@ -2630,6 +2631,9 @@ int main(int argc, char *argv[]) {
 
                 /* The efivarfs is now mounted, let's read the random seed off it */
                 (void) efi_take_random_seed();
+
+                /* Cache command-line options passed from EFI variables */
+                (void) cache_efi_options_variable();
```

Comment:
should be done only under the `!skip_setup` condition, otherwise we'll redo this needlessly on each daemon reexec.

(Note that the `efi_take_random_seed()` invocation got that wrong, it should be guarded the same way. I fix that in #16142)

Reply:
Done. (I guess.)